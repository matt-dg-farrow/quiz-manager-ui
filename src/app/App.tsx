import React from "react";
import "./App.css";
import LoginPage from "../features/loginPage";
import Header from "../components/header";
import Footer from "../components/footer";
import "../styles/all.scss";

function App() {
  return (
    <div className="App">
      <Header />
      <LoginPage />
      <Footer />
    </div>
  );
}

export default App;
