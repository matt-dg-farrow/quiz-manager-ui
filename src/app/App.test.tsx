import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import App from "./App";

describe("The Header component", () => {
  const wrapper = shallow(<App />);

  it("should contain the correct number of components", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
