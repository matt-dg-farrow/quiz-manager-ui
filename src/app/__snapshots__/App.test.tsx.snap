// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header component should contain the correct number of components 1`] = `
<div
  className="App"
>
  <Header />
  <LoginPage />
  <Footer />
</div>
`;
